google-cloud-testing/spanner-benchwrapper/src/main/proto/spanner.proto

```diff
@@ -18,48 +18,57 @@ package spanner_bench;
 option java_multiple_files = true;
 option java_package = "com.google.cloud.benchwrapper";
 
-message ReadQuery{
-  // The query to use in the read call.
-  string Query = 1;
+message Singer {
```

Comment:
Not sure why you had to update the protos here, so I won't provide feedback on this part of the code

Reply:
It's because I changed the protos on google3 that are used by the benchmarker tests to use with the new/real emulator, so these ones need to match. Album/Singer are the same as what we use in examples too.